pkg/p2p/libp2p/internal/blocklist/blocklist_test.go

```diff
@@ -33,12 +33,12 @@ func TestExist(t *testing.T) {
 	}
```

Comment:
Would it make sense to have some tests which asserts that blocklisted peer will have expected reason?